exercises/list-ops/canonical-data.json

```diff
@@ -222,13 +282,67 @@
         {
           "uuid": "be396a53-c074-4db3-8dd6-f7ed003cce7c",
           "description": "direction dependent function applied to non-empty list",
+          "comments": [
+            "Expects integer division (expects / to discard fractions).       "
+          ],
           "property": "foldr",
           "input": {
             "list": [2, 5],
             "initial": 5,
             "function": "(x, y) -> x / y"
```

Comment:
note for posterity: The possible ambiguous interpretation for this function were:

* acc, el -> acc / el
    * acc: 5, el: 5. new acc 1
    * acc: 1, el: 2. new acc 0
* acc, el -> el / acc
    * acc: 5, el: 5. new acc 1
    * acc: 1, el: 2. new acc 2
 
So clearly the **latter** was intended. Note that this is different from the other case, where the **former** was intended. The fact that these two same strings require two different interpretations is what's confusing and is the motivation for this PR.